src/MIDebugEngine/AD7.Impl/AD7PendingBreakpoint.cs

```diff
@@ -354,7 +360,7 @@ internal async Task BindAsync()
                     }
                     if (bindResult.BoundBreakpoints == null || bindResult.BoundBreakpoints.Count == 0)
                     {
-                        this.SetError(new AD7ErrorBreakpoint(this, bindResult.ErrorMessage), true);
+                        this.SetError(new AD7ErrorBreakpoint(this, bindResult.ErrorMessage, enum_BP_ERROR_TYPE.BPET_GENERAL_ERROR), true);
```

Comment:
If this is where you are seeing hardware breakpoint failures: we may need some sort of hack here where we look at words in the message to guess the importance (yucky, but I can't think of anything better). Otherwise, I think you want to just take out the change.

---
In reply to: [605125034](https://github.com/microsoft/MIEngine/pull/1123#discussion_r605125034) [](ancestors = 605125034)